VideoLocker/src/main/java/org/edx/mobile/view/CourseCombinedInfoFragment.java

```diff
@@ -345,23 +336,22 @@ private void populateFacePile(){
 
     }
 
-    private View generateAnnouncementView(AnnouncementsModel model){
-
-        View convertView = inflater.inflate(R.layout.row_announcement_list, null);
-
-        TextView date = (TextView) convertView.findViewById(R.id.announcement_date);
-        TextView content = (TextView) convertView.findViewById(R.id.announcement_content);
-
-        date.setText(model.getDate());
-
-        Spanned text = Html.fromHtml(model.content);
+    private void populateAnnouncements(List<AnnouncementsModel> announcementsList) {
+        if(announcementsList !=null && announcementsList.size()>0) {
+            hideEmptyAnnouncementMessage();
 
-        Spanned interceptedLinks = OutboundUrlSpan.interceptAllLinks(text);
-        content.setText(interceptedLinks);
-        content.setMovementMethod(LinkMovementMethod.getInstance());
+            final String divider = "<div style='height:2px; width:100%; background-color: #E2E3E5'></div>";
```

Comment:
Should be setting a meta viewport tag on this content? per https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag